tests/config/test_data.config

```diff
@@ -216,6 +216,8 @@ params {
 
                 cutandrun_bedgraph_test_1                     = "${test_data_dir}/genomics/homo_sapiens/illumina/bedgraph/cutandtag_h3k27me3_test_1.bedGraph"
                 cutandrun_bedgraph_test_2                     = "${test_data_dir}/genomics/homo_sapiens/illumina/bedgraph/cutandtag_igg_test_1.bedGraph"
+
+                test_rnaseq_vcf                               =  "${test_data_dir}/genomics/homo_sapiens/illumina/vcf/test.rnaseq.vcf"
```

Comment:
did you add test data in this PR or in another PR for this tool? Did the existing testdatasets not work?